content/en/containers/kubernetes/data_collected.md

```diff
@@ -86,12 +119,41 @@ Note that `kubernetes_state.*` metrics are gathered from the `kube-state-metrics
 
 ### Kubelet
 
+For more information, see the documentation for the [Kubelet][1] integration.
+
 {{< get-service-checks-from-git "kubelet" >}}
 
-### Kubernetes state
+### Kubernetes controller manager
+
+For more information, see the documentation for the [Kubernetes controller manager][2] integration.
+
+{{< get-service-checks-from-git "kube_controller_manager" >}}
+
+### Kubernetes metrics server
 
-{{< get-service-checks-from-git "kubernetes_state" >}}
+For more information, see the documentation for the [Kubernetes metrics server][4] integration.
+
+{{< get-service-checks-from-git "kube_metrics_server" >}}
+
+### Kubernetes scheduler
+
+For more information, see the documentation for the [Kubernetes scheduler][5] integration.
+
+{{< get-service-checks-from-git "kube_scheduler" >}}
+
+### Kubernetes state metrics core
+
+For more information, see the documentation for the [Kubernetes state metrics core][6] integration.
```

Comment:
It looks like there are a lot of service checks listed on the linked doc:
https://docs-staging.datadoghq.com/cswatt/k8s-data-collected/integrations/kubernetes_state_core/?tab=helm#service-checks
Is it expected that none of those service checks would be rendered on this page?

Reply:
thanks for catching this!